react/components/ProductImages/components/InnerZoom.js

```diff
@@ -28,22 +28,45 @@ export default class InnerZoom extends Component {
     },
   }
 
+  constructor(props){
+    super(props)
+    this.imageZoomed = React.createRef()
+    this.contentDiv = React.createRef()
+  }
+
+  getScale() {
+    let scale = {
+      x : 1,
+      y : 1
+    }
+
+    if(this.contentDiv.current && this.imageZoomed.current) {
```

Comment:
Your code style is not following the eslint style, after the if statement there must be a blank space and when you are using arithmetic operators the operands and the operators must be separated by a blank space too.